_data/tagging-status.yml

```diff
@@ -1722,13 +1722,13 @@
 
  - name: changebar
    type: package
-   status: unknown
+   status: partially-compatible
    included-in: [tlc3, arxiv01]
    priority: 2
    issues:
-   tests: false
-   tasks: needs tests
-   updated: 2024-07-14
+   tests: true
+   tasks: "Needs luatex85 loaded with lualatex."
```

Comment:
current tasks should go into comments and instead there it should say: "determine how to tag changed parts"

Reply:
Fixed